test/unit/configuration_test.rb

```diff
@@ -6,7 +6,7 @@
 
 module Packwerk
   class ConfigurationTest < Minitest::Test
-    include FixtureHelper
+    include RailsFixtureHelper
```

Comment:
Love this new commit!

It makes the dependencies on Rails more obvious. We can now see that `Configuration` needs `Rails` around to work, and we should change that (obviously in a separate PR).

Reply:
Here's the [separate PR](https://github.com/Shopify/packwerk/pull/68)